backend/models/feedbackModel.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+
+const feedbackSchema = mongoose.Schema(
+  {
```

Comment:
It might be good to include the date of when the feedback was created as well

Reply:
Fixed: Changes are reflected in recent commits